### DISCUSSION TITLE
Why are there a few advertising sentences appended to the output content?

### DISCUSSION THREAD

**Discussion by xxxzsgxxx** [ggml-org/whisper.cpp#1328] [General]
After performing speech recognition on an audio file using a medium or large model, I successfully obtained the transcribed text with timestamps for the spoken content. However, there is a certain probability that one or two advertising sentences will appear at the end. I have confirmed that these advertising sentences are not present in the audio file. Can anyone explain why this is happening and if anyone has encountered a similar situation, please provide the reason.

**Comment by RndyP** [comment]
What are they selling? :) Sorry, couldn't resist asking that.
Do you get these ads on background noise? Whisper is a large "loose" training set. Apparently, they used subtitled audio from all kinds of sources, so maybe they forgot to eliminate the commercials before they ran it through. Perhaps if it's noise, Whisper found the most likely match on an ad phrase.

**Reply by xxxzsgxxx** [reply]
I cannot hear the audio content of the advertisements. The ads in the transcribed text after ASR processing are usually subscription services that welcome subscriptions or encourage visiting certain services. I also suspect that certain content may trigger the whisper  “illusion."
It is quite strange.

**Reply by janngobble** [reply]
I know what you are talking about. Things like "Translation by XXX" or "Film by YYY", right? 

If so, generally that is because whisper is hallucinating, I think. I see it a lot - especially at the end of my (mostly movie and tv) content. The phrase does not appear in the soundtrack, hence I think it's an hallucination.
